documentation/overview/account_setup/connect-access.md

```diff
@@ -3,33 +3,89 @@ ospool:
   path: overview/account_setup/connect-access.md
 ---
 
-# Log In to "OSG Connect" Access Points
+# Log In to `uc.osg-htc.org` Access Points
 
 **This guide is for users who were notified by a member of the OSG team that they 
 will be using the "OSG Connect" Access Points. Do not go through the steps of this 
 guide until advised to by a Research Computing Facilitator**
 
-To join and use the "OSG Connect" Access Points (`ap20.uc.osg-htc.org`, 
-`ap21.uc.osg-htc.org`), you will go through the following steps: 
+> To use the "OSG Connect" Access Points (`ap20.uc.osg-htc.org`, 
```

Comment:
Do we still want to refer to them as "OSG Connect" (seeing as you took it out of the title)? Or both.

Reply:
As discussed live, removed "OSG Connect" from heading to make parallel structure with the UW access points.
But keeping "OSG Connect" in the paragraph as a clue, in case they don't recognize the address.
Lead to broader discussion about "codenames" for UW/UC access point groupings.